modules/repository-azure/build.gradle

```diff
@@ -190,11 +190,6 @@ tasks.named("thirdPartyAudit").configure {
     'org.bouncycastle.cert.X509CertificateHolder',
     'org.bouncycastle.cert.jcajce.JcaX509CertificateHolder',
     'org.bouncycastle.cert.jcajce.JcaX509v3CertificateBuilder',
-    'org.bouncycastle.crypto.InvalidCipherTextException',
```

Comment:
These were part of `bcprov-jdk18on` `1.77` but no longer used in `1.79` (updated transitive dependency from `nimbus-jose-jwt`) and therefore they don't need to be declared as missing classes here anymore.